packages/checkbox-react/example/index.tsx

```diff
@@ -3,25 +3,26 @@ import { initTabListener } from "@fremtind/jkl-core";
 import "@fremtind/jkl-core/core.css";
 import React from "react";
 import ReactDOM from "react-dom";
-import { Checkbox } from "../src";
+import { Checkbox, CheckboxGroup } from "../src";
 
 initTabListener();
 
 const App = () => {
     const [isCool, changeCoolness] = React.useState(false);
     return (
         <>
-            <Checkbox>I am checkbox!</Checkbox>
-            <Checkbox>Am I?</Checkbox>
-            <Checkbox checked={true}>I will never change</Checkbox>
-            <Checkbox checked={isCool} onChange={() => changeCoolness(!isCool)}>
+            <Checkbox name="checkbox1" onChange={(name, value) => alert(`${name} is now ${value}`)}>
```

Comment:
I really don't understand why we use `alert`? I think it is super annoying! The same applies for the portal.